pymc4/gp/mean.py

```diff
@@ -32,9 +43,9 @@ class MeanAdd(Mean):
 
     Parameters
     ----------
-    mean1 : {callable, pm.Mean}
+    mean1 : {callable, pymc4.gp.Mean}
```

Comment:
I think is ok to use `pm`

Reply:
I will check that out. I think it doesn't affect the doc tests by pydocstyle. So any name would work.